tests/Sinch.Tests/Voice/VoiceClientTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using FluentAssertions;
+using Sinch.Voice;
+using Sinch.Voice.Callouts;
+using Xunit;
+
+namespace Sinch.Tests.Voice
+{
+    public class VoiceClientTests
+    {
+        [Fact]
+        public void InitVoiceWithGlobalRegion()
+        {
+            var client = new SinchClient("id", "secret", "id");
+            var voiceClient = client.Voice("key", "secret");
```

Comment:
This is highlighting a pending question about unified authentication way for SDK in front of multiple APIs
Here, the voiceClient usage with dedicated key/secret (vs the ones used for unified SinchClient) is highlighting differences and not an unified way

Reply:
I'm confused. The clients are different. Can you elaborate what is expected, and what should we do to procede with review?